ui/app/css/itcss/components/sections.scss

```diff
@@ -1,426 +0,0 @@
-  visibility: hidden;
-}
-
-.editing-label {
-  display: flex;
-  justify-content: flex-start;
-  margin-left: 50px;
-  margin-bottom: 2px;
-  font-size: 11px;
-  text-rendering: geometricPrecision;
-  color: #f7861c;
-}
-
-.name-label:hover .edit-text {
-  visibility: visible;
-}
-
-/* tx confirm */
-
-.unconftx-section input[type=password] {
-  height: 22px;
-  padding: 2px;
-  margin: 12px;
-  margin-bottom: 24px;
-  border-radius: 4px;
-  border: 2px solid #f3c83e;
-  background: #faf6f0;
-}
-
-/* Info screen */
-.info-gray {
-  text-transform: uppercase;
-  color: $silver-chalice;
-}
-
-.icon-size {
-  width: 20px;
-}
-
-.info {
-  padding-bottom: 10px;
-  display: inline-block;
-  padding-left: 5px;
-}
-
-/* buy eth warning screen */
-.custom-radios {
-  justify-content: space-around;
-  align-items: center;
-}
-
-.custom-radio-selected {
-  width: 17px;
-  height: 17px;
-  border: solid;
-  border-style: double;
-  border-radius: 15px;
-  border-width: 5px;
-  background: rgba(247, 134, 28, 1);
-  border-color: #f7f7f7;
-}
-
-.custom-radio-inactive {
-  width: 14px;
-  height: 14px;
-  border: solid;
-  border-width: 1px;
-  border-radius: 24px;
-  border-color: $silver-chalice;
-}
-
-.radio-titles {
-  color: rgba(247, 134, 28, 1);
-}
-
-.eth-warning {
-  transition: opacity 400ms ease-in, transform 400ms ease-in;
-}
-
-.buy-subview {
-  transition: opacity 400ms ease-in, transform 400ms ease-in;
-}
-
-.input-container:hover .edit-text {
-  visibility: visible;
-}
-
-.buy-inputs {
-  font-size: 13px;
-  height: 20px;
-  background: transparent;
-  box-sizing: border-box;
-  border: solid;
-  border-color: transparent;
-  border-width: 0.5px;
-  border-radius: 2px;
-}
-
-.input-container:hover .buy-inputs {
-  box-sizing: inherit;
-  border: solid;
-  border-color: #f7861c;
-  border-width: 0.5px;
-  border-radius: 2px;
-}
-
-.buy-inputs:focus {
-  border: solid;
-  border-color: #f7861c;
-  border-width: 0.5px;
-  border-radius: 2px;
-}
-
-.activeForm {
-  background: #f7f7f7;
-  border: none;
-  border-radius: 8px 8px 0 0;
-  width: 50%;
-  text-align: center;
-  padding-bottom: 4px;
-}
-
-.inactiveForm {
-  border: none;
-  border-radius: 8px 8px 0 0;
-  width: 50%;
-  text-align: center;
-  padding-bottom: 4px;
-}
-
-.ex-coins {
-  text-transform: uppercase;
-  text-align: center;
-  font-size: 33px;
-  width: 118px;
-  height: 42px;
-  padding: 1px;
-  color: #4d4d4d;
-}
-
-.marketinfo {
-  color: $silver-chalice;
-  font-size: 15px;
-  line-height: 17px;
-}
-
-#fromCoin::-webkit-calendar-picker-indicator {
-  display: none;
-}
-
-#coinList {
-  width: 400px;
-  height: 500px;
-  overflow: scroll;
-}
-
-.icon-control .fa-sync {
-  visibility: hidden;
-}
-
-.icon-control:hover .fa-sync {
-  visibility: visible;
-}
-
-.icon-control:hover .fa-chevron-right {
-  visibility: hidden;
-}
-
-.inactive {
-  color: $silver-chalice;
-}
-
-.inactive button {
-  background: $silver-chalice;
-  color: $white;
-}
-
-.qr-ellip-address,
```

Comment:
Nit: You've removed the need for this class with the changes to `ReadOnlyInput`, but the class itself remains in-use despite having no associated rules. The same goes for `ellip-address` as well.

Reply:
good catch. follow up, there are some tests that rely on this class. 

https://github.com/MetaMask/metamask-extension/blob/d0366ad8f21bdded8d0ede99d3bbb72bc06529ba/test/e2e/ethereum-on.spec.js#L91-L100

While it would probably be fine to have them look for the class on ReadOnlyInput, the current selector code on that test is brittle in that if there were ever two of those in a view it wouldn't be able to differentiate.

Would you rewrite the selectors to be more discriminate or do you think it's safe to just look for the ReadOnlyInput className

Reply:
i opted to just use the .readonly-input__input class for now. if it becomes a problem in the future we can tackle it then.

Reply:
That sounds like a good enough solution for the time being.